src/vario/ui/settings/settings.h

```diff
@@ -266,6 +267,12 @@ class Settings {
 
   void toggleBoolNeutral(bool* boolSetting);
   void toggleBoolOnOff(bool* switchSetting);
+
+ private:
+  float sinkAlarmOptions_[2][11] = {
```

Comment:
No need for this to be a field of every instance of the class -- instead, I would move this to the .cpp file (ideally inside an anonymous namespace) and:

```cpp
namespace {
  constexpr float SINK_ALARM_OPTIONS[][11] = {
      {0, -1.2, -1.4, -1.6, -1.8, -2.0, -2.5, -3.0, -4.0, -5.0, -6.0},   // m/s
      {0, -240, -280, -320, -360, -400, -500, -600, -800, -1000, -1200}  // fpm
  };
}
```
(the anonymous namespace makes sure it's not accessible outside the file through some interesting edge case)